pkg/isb/stores/jetstream/reader.go

```diff
@@ -339,13 +334,16 @@ func (o *offset) AckIt() error {
 	return nil
 }
 
-func (o *offset) NoAck() {
+func (o *offset) NoAck(ctx context.Context) {
 	if o.cancelFunc != nil {
 		o.cancelFunc()
 	}
 	// Ignore the returned error as the worst case the message will
 	// take longer to be redelivered.
-	_ = o.msg.Nak()
+	log := logging.FromContext(ctx).With("Offset", o.String())
+	if err := o.msg.Nak(); err != nil {
+		log.Debug("Failed to nak the message")
```

Comment:
this should not be in Debug log level, Error level?, and also please print the error message too.

@whynowy adding log here has value?, since we have to get ctx into this.

Reply:
Letting `offset.NoAck()` return an error, and get the logged by `reader.NoAck()` might be better.

Reply:
Makes sense, updated.